src/backend/Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 flask = "*"
-"psycopg2-binary" = "*"
+"psycopg2" = "*"
```

Comment:
Lol. I personally prefer `psychopg2` and would like this to silently go through but I'll link https://github.com/codeforpdx/recordexpungPDX/pull/945 for context.

Reply:
@wittejm Ideally, you could just give us a dictatorial final verdict here so we won't waste more time on this.

Reply:
sure, `psycopg2` sounds preferable and the only reason to have `psycopg2-binary` was to support a non-essential developer script.

Reply:
my thoughts on this: the non-binary package works great in the container and is recommended best practice, running util scripts should generally happen *in the container*, and going outside of that is "advanced". the former docker configuration was rife with cross-platform peril, leading to workarounds like this. my intent with this branch is to alleviate the need for such things.